tests/training/trainer_test.py

```diff
@@ -755,6 +757,31 @@ def test_trainer_can_log_learning_rates_tensorboard(self):
 
         trainer.train()
 
+    def test_sanity_check_callback(self):
+        class BiasBatchNormModel(Model):
+            def __init__(self, use_bias=True):
```

Comment:
This has the `use_bias` parameter, but it's always `True`. Did you mean to parameterize this test to test both cases?

Reply:
Wait, this class is here twice. Can you re-use it from the other file?